libraries/common/effekt.effekt

```diff
@@ -156,6 +156,16 @@ type Ordering {
   Greater()
 }
 
+def compareInt(n: Int, m: Int) =
+  if (n == m) Equal()
+  else if (n < m) Less()
+  else Greater()
+
+def compareString(left: String, right: String) =
+  if (left == right) Equal()
+  else if (left < right) Less()
+  else Greater()
```

Comment:
This doesn't work on LLVM, hence #748. Moreover, it's probably worth it to have a few of these `compare` functions prepared for more types than just `Int`s and `String`s, but that's future work :)